depends/funcs.mk

```diff
@@ -157,48 +158,60 @@ $(1)_autoconf += LDFLAGS="$$($(1)_ldflags)"
 endif
 endef
 
+shortdate="+%H:%M:%S"
+
 define int_add_cmds
-$($(1)_fetched):
+$($(1)_introduced):
+	$(AT)echo "============================================================"
```

Comment:
No need for the "$(AT)"s for the echos.

A line starting with $(AT) means that it will be echoed to the terminal if you use ```make V=1```. For example, the mkdir/rm -f below aren't interesting, so we don't usually print them. But they can be shown for debugging with V=1.

Reply:
Hm, got it. I'll look into undoing this if `V=1`.

Reply:
Oddly, when I removed the $(AT)'s I got the opposite result:
```
$ make HOST=x86_64-apple-darwin11
echo "============================================================"
============================================================
echo "   Package : native_biplist"
   Package : native_biplist
echo "   Date    : `date`"
   Date    : Wed Jul 18 14:55:22 JST 2018
echo "============================================================"
============================================================
14:55:22: Extracting native_biplist...
14:55:22: Preprocessing native_biplist...
[...]
```

Reply:
@kallewoof Sorry, I realize that was very misleading.

An "@" in front of a rule causes it to not be echoed.
 ```$(AT)``` means "echo only if V=1" (otherwise it expands to "@").

Because we never want to echo the echo command itself, you can just use ```@echo foo``` rather than ```$(AT) echo foo```.

Reply:
@theuni Ahh... thanks, I didn't know that. I really need to brush up on my makefile skills one of these days...